.cirrus.yml

```diff
@@ -0,0 +1,54 @@
+check_android_task:
+  name: Run Android instrumented tests
+  env:
+    SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspressample/files/screenshots
+    TUTORIAL_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_SCREENSHOTS_ROOT_DIR: /data/data/com.kaspersky.kaspresso.tutorial/files/screenshots
+    ALLURE_SAMPLE_FAIL_SCREENSHOTS_ROOT_DIR: /data/user/0/com.kaspersky.kaspresso.alluresupport.sample/files/screenshots
```

Comment:
As far as I remember allure stores all screenshots including ones with failure in one dir